src/docs/product/discover-queries/query-builder.mdx

```diff
@@ -24,14 +24,14 @@ From the **Discover** page, you can build a query in three ways.
 
 There are four main building blocks that impact the results of your saved query. You can use a combination of these to narrow down your search.
 
-1. Global Selection Header
+1. Proejct, environment, and date range filters
 2. Search Conditions
 3. Interactive Graph
 4. Table Columns
 
-## Filter by Global Selection Header
+## Filter by project, enviornment, and date range
```

Comment:
Fixing capitalization and adding header redirect to avoid 404s

```suggestion
## Filter by Project, Environment, and Date Range {#filter-by-global-selection-header}
```